lib/scrabble.rb

```diff
@@ -1,7 +1,31 @@
 class Scrabble
 
   def score(word)
-    1
+    word = usable_word(word)
+    single_multipliers = Array.new(word.length, 1)
+    score_with_multipliers(word, single_multipliers)
```

Comment:
I have mixed feelings about this attempt to DRY out your code. This is something I might actually do. The other side is that it complicates the execution. Makes the code work harder than it has to. Just a thought. Teach the controversy.

Also, I'd probably do `single_multipliers = [1] * word.length`. Little syntax trick if you weren't aware.